pepper-apis/src/main/java/org/broadinstitute/ddp/util/Auth0Util.java

```diff
@@ -300,18 +297,6 @@ public static ManagementAPI getManagementApiInstanceForUser(String userGuid, Han
         return new ManagementAPI(auth0Domain, mgmtToken);
     }
 
```

Comment:
Most of the changes here are either 1) replacing calls with retry-enabled ones, or 2) moving functionality to `Auth0ManagementClient` and re-implementing with retries.